DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -1956,6 +1959,123 @@ function string RemapBingoEvent(string eventname)
 
 }
 
+static function GetBingoFailedGoals(DXRando dxr, string eventname, out string failed1, out string failed2)
+{
+    failed1 = "";
+    failed2 = "";
+
+    // keep in mind that a goal can only be marked as failed if it isn't already marked as completed
+    switch (eventname) {
+        case "JuanLebedev_Dead":
+            failed1 = "LebedevLived";
+            break;
+        case "Aimee_Dead":
+        case "LeMerchant_Dead":
+            failed1 = "AimeeLeMerchantLived";
```

Comment:
missing break statement, which makes me think this should be refactored to use returns

```suggestion
            failed1 = "AimeeLeMerchantLived";
            break;
```